fastestimator/util/util.py

```diff
@@ -214,6 +215,36 @@ def _custom_tf_print(*args, **kwargs):
     print_v2(*args, **kwargs)
 
 
+def validate_file(file_path: str) -> bool:
+    """Validate whether file valid or not.
+
+    Args:
+        file_path (str): location of the input file.
+
+    Returns:
+        bool: Whether the is valid.
+    """
+    try:
+        if Path(file_path).suffix == '.zip':
+            import zipfile
+            zip_file = zipfile.ZipFile(file_path)
+            _ = zip_file.namelist()
+        elif Path(file_path).suffix == '.gz':
+            if file_path.endswith('.tar.gz'):
+                import tarfile
+                with tarfile.open(file_path) as img_tar:
+                    _ = img_tar.getmembers()
+            else:
+                import gzip
+                f = gzip.open(file_path, 'rb')
+                _ = f.read()
+        elif not (os.path.exists(file_path) and os.path.getsize(file_path) > 0):
+            return False
+        return True
+    except Exception as e:
```

Comment:
could drop the e or maybe do 'Exception as _' to avoid this?

Reply:
Doing Exception as _ is raising codacy errors.